tools/Documentation/api-documentation/tankoubon-api.md

```diff
@@ -166,26 +174,29 @@ Name of the Category.
 {% endswagger-response %}
 {% endswagger %}
 
-{% swagger baseUrl="http://lrr.tvc-16.science" path="/api/tankoubons/:id/archive" method="put" summary="🔑Update a Tankoubon" %}
+{% swagger baseUrl="http://lrr.tvc-16.science" path="/api/tankoubons/:id" method="put" summary="Update a Tankoubon" %}
 {% swagger-description %}
-Modify a Tankoubon using an ordered array that can add new files, remove files, and update order.
+Update a Tankoubon.
 {% endswagger-description %}
 
 {% swagger-parameter name="id" type="string" required="true" in="path" %}
 ID of the Tankoubon to update.
 {% endswagger-parameter %}
 
-{% swagger-parameter name="archives" type="array" required="true" in="body" %}
-Ordered array with the IDs of the archives.
+{% swagger-parameter name="archives" type="json" required="true" in="body" %}
+Json with 2 keys "archives" and "metadata" defining:
+
+- archives: Ordered array with the IDs of the archives.
```

Comment:
Should we mention that this key is technically optional?  
It is possible to supply the API with just a metadata JSON if they only need to do a title/summary swap.  

(the reverse isn't really possible, but they can just add an empty `metadata` JSON which might be helpful to mention as well)

Reply:
Both actions have a piece of code like this, which ensures that if you don't send the key, then it doesn't do anything, so both are mutually optional.

```
if ( not defined $data->{"archives"} ) {
    return ( 1, "" );
}
```